main.py

```diff
@@ -157,3 +158,15 @@ def update_topology(self, event):
         switches = list(topology.switches.keys())
         links = list(topology.links.keys())
         log.debug(f"Topology graph updated with switches: {switches}, links: {links}.")
+
+    def _get_latest_topology(self):
+        """Get the latest topology from the topology napp."""
+        try:
+            topology_napp = self.controller.napps[("kytos", "topology")]
+        except KeyError:
+            log.debug("Failed to get topology napp for forcing topology update.")
```

Comment:
Thinking more about the logged level, I'd say this even deserves to be a `log.warning`. Although it's just a fallback to update the topo, `topology` is already a dependency of this NApp, so it's not being loaded could even be an error or warning in this case since it's a fallback mechanism. Do you see what I mean? And if someone made a mistake of loading `topology` after `pathfinder` in the wrong order, then the warning message would also be helpful when it still not loaded.